include/Dialect/LWE/IR/LWEAttributes.td

```diff
@@ -30,6 +30,16 @@ class LWE_EncodingAttrWithScalingFactor<string attrName, string attrMnemonic, li
   );
 }
 
+class LWE_UnspecifiedEncodingAttr<string attrName, string attrMnemonic, list<Trait> traits = []>
```

Comment:
Why have a parent class here?

Reply:
This was to anticipate an RLWE unspecified encoding, but i'll defer to when its needed :) Removed.